### PR TITLE
fix: clear the app-termination observer when we dealloc

### PR DESCRIPTION
**Pull request by daytime-em**

### CODE REVIEW

File: MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m [hunk 1]
```diff
@@ -210,13 +209,6 @@ - (void)handleDidPlayToEndTimeNotification:(NSNotification *)notification {
     }
 }
 
-- (void)handleApplicationWillTerminate:(NSNotification *)notification {
```
Comment (andrewjl-mux) [inline MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m:213]: we still want this in right?

Comment (andrewjl-mux) [reply MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m:213]: never mind, I see it's included twice

**Review by andrewjl-mux — APPROVED** [review]

**Review by andrewjl-mux — APPROVED** [review]
good catch

**Review by cjpillsbury — APPROVED** [review]
🚢 🇮🇹

**Pull request merged**